nixos/modules/hardware/corectrl.nix

```
@@ -12,6 +12,16 @@ in
       Add your user to the corectrl group to run corectrl without needing to enter your password
     '');
 
+    package = mkOption {
+      type = types.package;
+      default = pkgs.corectrl;
+      description = lib.mdDoc ''
+        The `corectrl` derivation to use. Useful for
+        overriding the configuration options used
+        for the package.
+      '';
+    };
```

```suggestion
    package = mkPackageOption pkgs "corectrl" { };
```

If you want to keep

> Useful for overriding the configuration options used for the package.

as well, use the following instead:

```suggestion
    package = mkPackageOption pkgs "corectrl" {
      extraDescription = "Useful for overriding the configuration options used for the package.";
    };
```

We usually prefer the use of the shorthand function `mkPackageOption` for `package` options over having to manually repeat the very same `mkOption` structure in way too many places in nixpkgs.

You also have a failing ci check because you didn't set `defaultText = literalExpression "pkgs.corectrl";`.

`mkPackageOption` on the other hand, just does this for you.

---

Done, thank you! I didn't know there was a shorthand function. That's so convenient!